app/application.js

```diff
@@ -1,26 +1,33 @@
 "use strict";
 
-var root = 'http://localhost:4000/api';
-var origin = window.location.origin;
+const root = 'http://localhost:4000/api';
+const origin = window.location.origin;
+
+function link(id, text){
+  return `<a href= ${origin}/announcement.html?id=${id}>${text}<br/><br/></a>`
+}
 
 var App = {
   init: function init() {
     $("body").prepend(
         `<a href = ${origin}/new.html> + </a>`)
   },
-  constableApi: class ConstableApi {
+  path: class Path {
```

Comment:
I'm not sure about how to best structure these but I don't think this should be part of `App` anymore. It's weird to ahve a property that returns a whole class here.